systemtests/local_user_test.go

```diff
@@ -103,6 +103,17 @@ func (s *systemtestSuite) userUpdate(c *C) {
 
 			// try login again using new password
 			_ = loginAs(c, username, "test")
+
+			// userUpdateEndpoint is not adminOnly any more. It can be accessed by user himself.
```

Comment:
Nit: this comment is not very helpful. Just make is say that it is admin or calling user. Recording history in comment is usually good only for a short time.